tools/bcftools_strip_ann.cwl

```diff
@@ -18,10 +18,11 @@ arguments:
     valueFrom: >-
       set -eo pipefail
 
-      bcftools annotate -x $(inputs.strip_info) $(inputs.input_vcf.path) -O z 
-      -o $(inputs.output_basename).$(inputs.tool_name).INFO_stripped.vcf.gz
-
-      tabix $(inputs.output_basename).$(inputs.tool_name).INFO_stripped.vcf.gz
+      (bcftools annotate -x $(inputs.strip_info) $(inputs.input_vcf.path) -O z 
+      -o $(inputs.output_basename).$(inputs.tool_name).INFO_stripped.vcf.gz &&
+      tabix $(inputs.output_basename).$(inputs.tool_name).INFO_stripped.vcf.gz) ||
+      (echo "Check errors, likely does not have INFO, trying to pass input instead >&2"; cp $(inputs.input_vcf.path) .;
```

Comment:
is the `>&2` functional here? it looks like it's trapped in the quotes

Reply:
Ah, yeah, I noticed it didn't seem to work. That makes sense. I can fix that while troubleshooting merge conflict